app/javascript/packages/components/process-list/process-list-heading.tsx

```diff
@@ -0,0 +1,25 @@
+import { forwardRef } from 'react';
+import type { ReactNode } from 'react';
+
+interface ProcessListHeadingProps extends Record<string, any> {
+  children?: ReactNode;
+
+  className?: string;
+
+  // pass unstyled to render unstyled text
+  unstyled?: boolean;
+}
+
+function ProcessListHeading({ children, className, unstyled }: ProcessListHeadingProps, ref) {
+  const headingClass = unstyled === true ? undefined : 'usa-process-list__heading';
+  const noClasses = headingClass === undefined && className === undefined;
+  const classes = noClasses ? undefined : [headingClass, className].filter(Boolean).join(' ');
```

Comment:
`.filter(Boolean)` would remove all the unset classes anyways, so it seems we could skip some logic here?

```suggestion
  const classes = [headingClass, className].filter(Boolean).join(' ');
```

Reply:
addressed in 0046960aac92a64d5598b0786adbd8cd7c524594